lib/jira_api/issue_types/new_project_request.py

```diff
@@ -15,14 +15,22 @@ def _create_properties(self) -> Dict:
         answers = proforma_form.value.state.answers
 
         properties["project_name"] = getattr(answers, "3").text
-        properties["users"] = getattr(answers, "29").adf.content[0].content[0].text
+        potential_users_info = getattr(answers, "29").adf.content
+        if potential_users_info != []:
+            properties["users"] = potential_users_info[0].content[0].text
+        else:
+            properties["users"] = ""
```

Comment:
An empty string evaulates to false so we can just use `not`.
```suggestion
        potential_users_info = getattr(answers, "29").adf.content
        if not potential_users_info:
            properties["users"] = potential_users_info[0].content[0].text
        else:
            properties["users"] = ""
```

Reply:
Also, at some point we should look at how we are expecting users to fill in this part of the form. Right now it just returns a paragraph of text,. How will we split that? Would we use a delimiter like `,` or `;`. Not for this PR